components/job-orchestration/job_orchestration/scheduler/query/query_scheduler.py

```
@@ -1105,39 +1144,42 @@ async def check_job_status_and_update_db(db_conn_pool, results_cache_uri):
-                    db_conn, search_job, returned_results, results_cache_uri
-                )
-            elif job_type in (QueryJobType.EXTRACT_JSON, QueryJobType.EXTRACT_IR):
-                await handle_finished_stream_extraction_job(db_conn, job, returned_results)
-            else:
-                logger.error(f"Unexpected job type: {job_type}, skipping job {job_id}")
+            with bound_contextvars(**_get_query_job_log_context_from_job(job)):
+                try:
+                    returned_results = try_getting_task_result(
+                        job.current_sub_job_async_task_result
+                    )
+                except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)